src/applications/hca/actions.js

```diff
@@ -98,10 +118,55 @@ export function getEnrollmentStatus(formData) {
       ) {
         callFake404(dispatch);
       } else {
-        callFakeSuccess(dispatch, HCA_ENROLLMENT_STATUSES.canceledDeclined);
+        callFakeSuccess(dispatch, HCA_ENROLLMENT_STATUSES.enrolled);
       }
     } else {
       callAPI(dispatch, formData);
     }
   };
 }
+
+export function getDismissedHCANotification() {
+  return dispatch => {
+    dispatch({ type: FETCH_DISMISSED_HCA_NOTIFICATION_STARTED });
+    const url = `/notifications/dismissed_statuses/form_10_10ez`;
+
+    apiRequest(
+      url,
+      null,
+      data =>
+        dispatch({ type: FETCH_DISMISSED_HCA_NOTIFICATION_SUCCEEDED, data }),
+      ({ errors }) =>
+        dispatch({ type: FETCH_DISMISSED_HCA_NOTIFICATION_FAILED, errors }),
+    );
+  };
+}
+
+export function setDismissedHCANotification(status, statusEffectiveAt, update) {
+  return dispatch => {
+    dispatch({
+      type: SET_DISMISSED_HCA_NOTIFICATION_STARTED,
+      data: statusEffectiveAt,
+    });
+    if (update) {
```

Comment:
Does the UI know if this should be an update or a post? That seems odd to me, but I'm not sure I understand the interface, since I couldn't find where this application creator is called.

Reply:
Yeah, there's going to be a component method like this that'll call this action creator:
```
this.props.setDismissedHCANotification(
  enrollmentStatus,
  enrollmentStatusEffectiveDate,
  !!this.props.dismissedNotificationDate
);
```
If we've got a `dismissedNotificationDate` already, then we need to call the PUT endpoint instead of the POST

Reply:
You could get that from the state in this action creator (there's a second parameter called `getState` after dispatch when you return a function), instead of passing it as a parameter. Just seems like an implementation detail that the React components shouldn't care about.